docs/en/contributing-to-appium/appium-from-source.md

```diff
@@ -18,26 +18,28 @@ instance of an Appium server, and then run your test.
 The quick way to get started:
 
 ```center
-$ git clone https://github.com/appium/appium.git
-$ cd appium
-$ ./reset.sh
-$ sudo ./bin/authorize-ios.js # for ios only
-$ node .
+git clone https://github.com/appium/appium.git
+cd appium
+npm install
+gulp transpile
+sudo ./bin/authorize-ios.js # for ios only
+node .
 ```
 
 ### Hacking on Appium
 
-Make sure you have ant, maven, adb installed and added to system PATH, also you
+Make sure you have `ant`, `maven`, `adb` installed and added to system `PATH`, also you
 would need the android-16 sdk (for Selendroid) and android-19 sdk installed.
 From your local repo's command prompt, install the following packages using the
-following commands (if you didn't install `node` using homebrew, you might have
-to run npm with sudo privileges):
+following commands (if you didn't install `node` using Homebrew, you might have
+to run `npm` with sudo privileges):
 
 ```center
 npm install -g mocha
-npm install -g grunt-cli
-node bin/appium-doctor.js --dev
-./reset.sh --dev
+npm install -g gulp
+node ./bin/appium-doctor.js --dev
```

Comment:
@imurchie has this changed now that it's no longer a part of the appium package itself?